jplag.frontend-utils/src/main/java/de/jplag/TokenPrinter.java

```diff
@@ -152,4 +182,16 @@ private static List<String> linesFromFile(File file) {
         }
         return Collections.emptyList();
     }
+
+    /**
+     * Returns the number of digits (including a minus) of the given number.
+     */
+    private static int digitCount(int index) {
```

Comment:
Rename index to number or something like that :)

Reply:
True that^^